app/src/ui/lib/join.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Insert a separator element between each entry of an array of elements
+ *
+ * @param elements
```

Comment:
Yo dawg, I meant to write some docs but totally forgot

Reply:
I'm not even sure we need the docs for the params, I added some anyways, but I'm fine with removing later if y'all are in agreement.